docs/tutorial/saxpy.rst

```diff
@@ -0,0 +1,780 @@
+.. meta::
+  :description: The SAXPY tutorial on HIP
+  :keywords: AMD, ROCm, HIP, SAXPY, tutorial
+
+*******************************************************************************
+Tutorial: SAXPY - Hello, HIP
+*******************************************************************************
+
+This tutorial explains the basic concepts of the single-source
+Heterogeneous-computing Interface for Portability (HIP) programming model and
```

Comment:
Can we change it to HIP (Heterogeneous-computing Interface) programming model. I think most people will call it HIP instead of its full name.

Reply:
This is defined by our documentation styling guide, and we are using the same at other locations. "The full text and then the acronym in parenthesis, and the following references use the acronym."

Example:
https://github.com/ROCm/HIP/blob/develop/docs/index.md